src/main/java/me/fril/modblocker/MBConfig.java

```diff
@@ -14,4 +14,12 @@
 	@Config.Name("Disconnection message")
 	@Config.Comment("Here you put what message the client recieves when kicked from the server, use '@MOD' to represent the offending modid")
 	public String disconnectMessage = "Mod: &s is not allowed on this server.";
+
+	@Config.Name("Ban message")
+	@Config.Comment("Here you put what message the client recieves when banned from the server")
+	public String banMessage = ""; //Todo : Add a message
+
+	@Config.Name("Action")
+	@Config.Comment("What to do if one blacklisted mod is detected ? [1 : disconnect, 2: BAN HAMMER]")
+	public int actionID = 1;
```

Comment:
You can actually make these the literal enums

Have a example https://github.com/Suffril/Weeping-Angels/blob/abcdd11b4fed519452e54adea0a38a890568e37b/src/main/java/me/fril/angels/config/WAConfig.java#L63-L66

Reply:
Nice, I'll look at that